pingcap/tidb
# server: initialize connectionInfo even if the audit plugins is not enabled

### What problem does this PR solve?

Issue Number: close #29038

Problem Summary:

### What is changed and how it works?

 Initialize connectionInfo even if the audit plugins is not enabled.

### Check List

Tests 


1. Deploy a cluster
2. Deploy a audit plugin
3. Create a session to tidb
4. Enable audit plugins
5. Execute some SQL commands
6. Check whether audit logs contain connection information.
![image](https://user-images.githubusercontent.com/46436429/140009162-cd12521c-9105-4758-ae60-a36dc75342d4.png)
there is no connection information
7. tiup cluster patch tidb
8. Create a session to tidb
9. Enable audit plugins
10. Execute some SQL commands
11. Check whether audit logs contain connection information
![image](https://user-images.githubusercontent.com/46436429/140009065-1ab15b8c-27e3-40ce-8241-51ee0beccbef.png)
there is connection information.



Side effects


Documentation


### Release note



```release-note
Fixed a bug where there was no connection information for existing sessions after the audit plugin was enabled
```
